### ISSUE TITLE
Simplify default urls

### ISSUE THREAD

**Issue by weslien** [weslien/playpal#3]
Remove uuid from url, replace with phone friendly format

**Comment by jensalm** [comment]
Alias functionality implemented in cmscore.

**Comment by weslien** [comment]
I still think it is a bad choice to have uuid's as default identifier.
for a url to be usable at all, you need to add an alias.

If, for example, we were to use unique integers (f.ex. from a db sequence) starting with 1000, it would make the content more accessible off the bat. Sure, urls like /leaf/12345 don't mean much, but they are more friendly on the eyes than /leaf/12dh6-jhdf7d-9874h4-jkshfdkjh-02j7s. Looks more like a serial number for photoshop, and those aren't easy to say over the phone either.

I say reopen this issue.

**Comment by jensalm** [comment]
Agreed! UUID is just a first step until we have a more sustainable solution (which requires a real db).
